.github/workflows/default.yaml

```diff
@@ -102,13 +102,10 @@ jobs:
       - name: Install prerequisite Linux packages
         if: runner.os == 'Linux'
         run: |
+          # required for "apt-get build-dep" to work
+          sudo sed --in-place -E 's/# (deb-src.*updates main)/  \1/g' /etc/apt/sources.list
           sudo apt-get --quiet=2 update
-          sudo apt-get --quiet=2 install libtool-bin
-          sudo apt-get --quiet=2 install libcppunit-dev
-          sudo apt-get --quiet=2 install libldap2-dev
```

Comment:
In my [tests](https://github.com/measurement-factory/squid/actions/runs/3228261796/jobs/5284123736), `apt-get build-dep` does not install libldap2-dev explicitly (in GitHub Actions environment), but it does install enough for Squid to detect LDAP support, removing the need for this temporary workaround -- Squid build `--with-ldap` succeeds without this line.

```
checking for LIBLDAP... yes
checking for ldap.h... yes
checking for lber.h... yes
checking for mozldap/ldap.h... no
checking for LDAP... 1
checking for OpenLDAP... 1
checking for Sun LDAP SDK... 0
checking for Mozilla LDAP SDK... 0
checking for LDAP_OPT_DEBUG_LEVEL... 1
checking for LDAP_SCOPE_DEFAULT... 1
checking for LDAP_REBIND_PROC... 1
checking for LDAP_REBINDPROC_CALLBACK... 0
checking for LDAP_REBIND_FUNCTION... 0
checking for LDAP_URL_LUD_SCHEME... 0
```

Reply:
Oh. I see Ubuntu is transitioning from libldap2-dev to libldap-dev...
```
Setting up libldap-dev:amd64 (2.5.13+dfsg-0ubuntu0.22.04.1) ...
```